Helper/Cart.php

```diff
@@ -1399,16 +1399,23 @@ public function getCartItems($quote, $storeId = null, $totalAmount = 0, $diff =
         $total = $quote->getTotals();
         if (isset($total['giftwrapping']) && ($total['giftwrapping']->getGwId() || $total['giftwrapping']->getGwItemIds())) {
             $giftWrapping = $total['giftwrapping'];
+
+
             $totalPrice = $giftWrapping->getGwPrice() + $giftWrapping->getGwItemsPrice() + $quote->getGwCardPrice();
             $product = [];
-            $product['reference']    = $giftWrapping->getGwId();
+            $gwId = $giftWrapping->getGwId();
+            $product['reference']    = $gwId;
             $product['name']         = $giftWrapping->getTitle()->getText();
             $product['total_amount'] = CurrencyUtils::toMinor($totalPrice, $currencyCode);
             $product['unit_price']   = CurrencyUtils::toMinor($totalPrice, $currencyCode);
             $product['quantity']     = 1;
             $product['sku']          = trim($giftWrapping->getCode());
             $product['type']         = self::ITEM_TYPE_PHYSICAL;
-
+            $objectManager = \Magento\Framework\App\ObjectManager::getInstance();
```

Comment:
The Magento\GiftWrapping\Model\Wrapping class is only available on EE, not CE. So we can't use dependency injection to initialize the class.  We just can use object manager. Same logic here https://github.com/BoltApp/bolt-magento2/blob/master/Helper/Cart.php#L2369 when initializing Magento\Reward\Model\Reward class